client-ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts

```diff
@@ -1,9 +1,16 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-import { IHubProtocol, ProtocolType, MessageType, HubMessage, InvocationMessage, ResultMessage, CompletionMessage, StreamInvocationMessage } from "./IHubProtocol";
-import { BinaryMessageFormat } from "./Formatters"
-import * as msgpack5 from "msgpack5"
+import { IHubProtocol, ProtocolType, MessageType, HubMessage, InvocationMessage, ResultMessage, CompletionMessage, StreamInvocationMessage } from "@aspnet/signalr";
+import { BinaryMessageFormat } from "./BinaryMessageFormat"
+import { Buffer } from 'buffer';
+import * as msgpack5 from "msgpack5";
+
+// This shim is needed to handle the fact that msgpack is exported like a CommonJS module
+// but we're building as an ECMAScript module.
+function msgpackshim(): msgpack5.MessagePack {
+    return <msgpack5.MessagePack>msgpack5.call({});
+}
```

Comment:
\*sigh* msgpack is goofy. I'm not quite sure why this was "working" before. It may not have been fully working. I get typescript warnings in VSCode with the original code. This adds some forced casting to clear those out. I'll re-review that before merge.